### ISSUE TITLE
archiving is failing

### ISSUE THREAD

**Issue by mjsibbald** [AndreaMelle/iosvr#2]
Althuogh building is working, when I want to archive it, it fails and says its missing hfgl/hfglVersion.h in th iosvr library.

I just cant figure it out.

**Comment by AndreaMelle** [comment]
Probably a different header search path for Release configuration.
The hfgl headers are a mess, I agree.
I will eventually fix it up, but please note this code is currently not
production ready or supported in any way, it's just a personal repo.

**Comment by mjsibbald** [comment]
Yes Andrea, I understand. But I've never worked with static libraries. So My project was failing and I remembered that yours was using it as well. So tested it and confirmed it was having the same problem. I fixed it by adding the hfgl as a dependency to the iosvr lib. So if you ever run into this problem, you know where to look. 

Never intented to ask for support, as its more like a general problem. (search on Stackoverflow drove me crazy). 

Love your work and there are very many nice things in there, especially the EKF and coremotion approach (as noted, latency is crap).
